library/ecjpake.c

```diff
@@ -559,6 +579,8 @@ int mbedtls_ecjpake_read_round_two( mbedtls_ecjpake_context *ctx,
     const unsigned char *end = buf + len;
     mbedtls_ecp_group grp;
     mbedtls_ecp_point G;    /* C: GB, S: GA */
+    ECJPAKE_VALIDATE_RET( ctx != NULL );
```

Comment:
Not a blocker: please leave a blank line after the block of declarations.

Reply:
Fixing.